pkg/triggers/watcher.go

```diff
@@ -41,13 +41,12 @@ type k8sInformers struct {
 	clusterEventInformers []coreinformerv1.EventInformer
 	configMapInformers    []coreinformerv1.ConfigMapInformer
 
-	testTriggerInformer testkubeinformerv1.TestTriggerInformer
-	testSuiteInformer   testkubeinformerv2.TestSuiteInformer
-	testInformer        testkubeinformerv3.TestInformer
+	testTriggerInformers []testkubeinformerv1.TestTriggerInformer
+	testSuiteInformers   []testkubeinformerv2.TestSuiteInformer
+	testInformers        []testkubeinformerv3.TestInformer
 }
 
-func newK8sInformers(clientset kubernetes.Interface, testKubeClientset versioned.Interface,
-	testkubeNamespace string, watcherNamespaces []string) *k8sInformers {
+func newK8sInformers(clientset kubernetes.Interface, testKubeClientset versioned.Interface, testkubeNamespace string, watcherNamespaces []string, watchTestkubeCrAllNamespaces bool) *k8sInformers {
```

Comment:
where do you use watchTestkubeCrAllNamespaces?

Reply:
My mistake. I meant to make it optional as some users might not want this functionality.

https://github.com/kubeshop/testkube/pull/4106/files#diff-59b1352b84890f4bab146e9c47bb21322ac161fd777114091c762d61838ca3efR68-R73